update/process/shardBlock_test.go

```diff
@@ -163,6 +163,7 @@ func TestCreateNewBlock(t *testing.T) {
 		ReceiptsHash: []byte("receiptHash"), TxCount: 2,
 		MetaBlockHashes: [][]byte{metaHash}, MiniBlockHeaders: expectedMbHeaders,
 		AccumulatedFees: big.NewInt(0),
+		DeveloperFees:   big.NewInt(0),
```

Comment:
DevFeesInEpoch is not needed to be set on 0 ?

Reply:
It is only a test